tests/imdocker-multi-line.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# This is part of the rsyslog testbench, licensed under ASL 2.0
+
+# imdocker unit tests are enabled with --enable-imdocker-tests
+. ${srcdir:=.}/diag.sh init
+
+generate_conf
+add_conf '
+template(name="outfmt" type="list") {
+  constant(value="HEADER ")
+  property(name="msg" format="json")
+  constant(value="\n")
+}
+
+module(load="../contrib/imdocker/.libs/imdocker" PollingInterval="1"
+        GetContainerLogOptions="timestamps=0&follow=1&stdout=1&stderr=0")
+action(type="omfile" template="outfmt"  file="'$RSYSLOG_OUT_LOG'")
+'
+
+#export RS_REDIR=-d
+startup
+NUM_ITEMS=1000
+# launch a docker runtime to generate some logs.
+# These logs started after start-up should get from beginning
+sleep 1
```

Comment:
FYI: `sleep` doesn't really work, as timing is very very different (and often slow) inside CI.